### ISSUE TITLE
Search box in Firefox does not get focus when popup opens

### ISSUE THREAD

**Issue by luckyrat** [kee-org/browser-addon#145]
This is a Firefox bug; it works fine in Chrome so will hopefully get fixed as soon as a new version of Firefox with the fix is released.

Blocked on https://bugzilla.mozilla.org/show_bug.cgi?id=1324255

(This refers to the unreleased search feature, due in Kee 2.2 in February 2018)

**Comment by luckyrat** [comment]
This will be resolved when Firefox 60 is released.

It's already working in Firefox Nightly.

**Comment by ggeorgg** [comment]
I can confirm, that it works on Firefox Nichtly 60.0a1 (2018-03-07) (64-Bit). 
I do have an additonal request. I am not sure if I should open a new issue for this: If I search for an entry and then click this entry, the website is just opened. I expected that the website is opened and the credentials of this entry are auto filled. I know the Kee setting to automatically fill in the credentials, but I can not use this, because I have several entries which do have the same url.

**Comment by ggeorgg** [comment]
Any news here?

**Comment by luckyrat** [comment]
Sorry @ggeorgg , I'm not likely to spot all new issues within a blocked issue. Please describe the problem you're trying to solve and any suggested solution on the [community forum](https://forum.kee.pm) and if appropriate, we can create a new GitHub issue following that discussion, potentially after gauging interest from other users. Thanks.

**Comment by ggeorgg** [comment]
Am I right that this one can be closed?

**Comment by luckyrat** [comment]
Yep thanks. This works correctly since Firefox 60.
